## **Hypothesis**
___

### **Null Hypothesis(H_0)**

- The mean of weekly operating cost is equal to the theoretical mean cost according to the cost model.which is oppsite of what restaurants owners are claiming

### **Alternative hypothesis(H_1)**

- The mean of weekly operating cost is greater than the theoretical mean cost according to the cost model.which is exactly what restaurant owner are claimed

___

### **Calculating the test statistic** 

- Calcualating the theoretical mean of weekly cost model using the provided data 

In [49]:
import numpy as np
import scipy.stats as stats

In [73]:
#Given data
sample_mean = 3050
X = 600
n = 25
#calculating theoretical mean from given data 
W = 1000 + 5*X

#calculating standard deviation 
sigma = 5*25
print("theoretical mean:",W)
print("sigma:",sigma)

theoretical mean: 4000
sigma: 125


In [79]:
#calculating test statistics
t = (sample_mean - W) / (sigma / np.sqrt(n))  

print("test stat :",t)

test stat : -38.0


### **Calculating critical value**

In [82]:
alpha = 0.05
critical_value = stats.norm.ppf(1 - alpha)
print("critical value:",critical_value)

critical value: 1.6448536269514722


### **Making The Decision**

 Making an appropriate decision by comparing the critical value(z) with test value(t).

  - test value (t) = -38.0
  - critical value (Z) = 1.645

 therefore the test value is smaller than the critical value ,we reject null hypotheses.

In [87]:
#making the decision based on the values of test statistic and critical value by comparing them
if t < critical_value:
    decision = "Reject the null hypothesis"
else:
    decision = "Fail to reject the null hypothesis"

## **Conclusion**

Based on the decision, we reject the null hypothesis. This provides strong evidence to support that the restaurant owner's of 25 restaurants claim that the weekly operating costs are higher than the what cost model suggests.

In [121]:
if t < critical_value : conclusion = "There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests."
else:
 conclusion = "There is not enough evidence to support the claim."

In [129]:
print(f'Test Statistic: {t}')
print(f'Critical Value: {critical_value}')
print(f'Decision: {decision}')
print(f'Conclusion: {conclusion}')

Test Statistic: -38.0
Critical Value: 1.6448536269514722
Decision: Reject the null hypothesis
Conclusion: There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.
